content/authorization/application-registration-prerequisites.md

```diff
@@ -0,0 +1,256 @@
+	
+
+## FAQ
+
+**Are these security requirements required of all Cerner Ignite applications?**
+
+The security requirements outlined in this document are currently only required for all Ignite applications targeting the patient persona. Though, Cerner encourages the use of these best practices for all end user facing applications.
+
+**Why are OV/EV certificates preferred by Cerner?**
+
+Organization-Validated (OV) and/or Extended-Validation (EV) certificates ensure that the certificate is being issued to an official business entity (with a name and address). This helps provide transparency to users (patients) about who they are sharing their data with.  Cerner does not perform validation of the incorporation of your business. By utilizing an OV or EV certificate, Cerner trusts that the certificate authority has performed such validation and will correspondingly display your organization's name in association with the application.
+
+**Is either an OV or EV certificate acceptable?**
+
+Yes, at the time of writing Cerner does not distinguish OV and EV certificates, so either is acceptable. However, some web browsers might give preferential treatment to EV certificates.
+
+**Are self-signed certificates acceptable?**
+
+No. Cerner does not support the use of self-signed certificates.
+
+**What Certificate Authorities (CAs) does Cerner trust?**
+
+Refer to [code TLS Guidance][code TLS Guidance] for a summary of the CAs that Cerner trusts.
+
+**How/when does Cerner use eTLD+1 values?**
+
+In some situations where Cerner cannot determine the business entity providing an application, Cerner falls back to using the domain name of the application's primary redirect URI to describe the application. For brevity/consistency, the eTLD+1 value will be used in these situations.
+
+**Why does Cerner only display the eTLD+1 portion of the domain name?**
+
+This is done to help avoid confusing users (patients). Google has a [good write-up explaining the security benefits of this practice](https://sites.google.com/a/chromium.org/dev/Home/chromium-security/enamel).
+
+**How do I get a Cerner client identifier for my application?**
+
+Cerner assigns your application a unique client identifier for your application as part of the registration process at the [code console][code console].  This Cerner client identifier is also referred to as a System Account ID in some Cerner Central tooling https://cernercentral.com/system-accounts/.
+
+**Why is a DNS TXT record recommended?**
+
+The DNS TXT record helps prove that the website intends to host your application. Cerner does not automatically directly assume that URLs provided by a developer are associated with the website operator. By publishing a DNS TXT record, Cerner can verify that the party that controls the website address intends to associate itself with the your registered application.
```

Comment:
"Cerner does not automatically directly assume" ... need "and" in there? automatically and directly?

Reply:
I removed the word "directly" to read "Cerner does not automatically assume that URLs provided by a developer are associated with the website operator."